src/foam/dao/EasyDAO.js

```diff
@@ -311,18 +311,26 @@ return delegate;
       name: 'syncProperty',
       generateJava: false
     },
+    {
+      name: 'retryBoxMaxAttempts',
+      generateJava: false,
+    },
     {
       /** Destination address for server. */
       name: 'serverBox',
       generateJava: false,
       factory: function() {
         // TODO: This should come from the server via a lookup from a NamedBox.
-        return this.SessionClientBox.create({ delegate: this.RetryBox.create({ delegate:
-          this.TimeoutBox.create({ delegate:
-          this.remoteListenerSupport ?
-              this.WebSocketBox.create({ uri: this.serviceName }) :
-              this.HTTPBox.create({ url: this.serviceName })
-        })})});
+        return this.SessionClientBox.create({
+          delegate: this.RetryBox.create({
+            maxAttempts: this.retryBoxMaxAttempts,
```

Comment:
If 0, couldn't we just not add the RetryBox?

Reply:
Updated